core/dbt/include/global_project/macros/materializations/models/table/create_table.sql

```diff
@@ -0,0 +1,16 @@
+{% macro create_table_as(temporary, relation, sql) -%}
+  {{ adapter.dispatch('create_table_as', 'dbt')(temporary, relation, sql) }}
+{%- endmacro %}
+
+{% macro default__create_table_as(temporary, relation, sql) -%}
+  {%- set sql_header = config.get('sql_header', none) -%}
+
+  {{ sql_header if sql_header is not none }}
+
+  create {% if temporary: -%}temporary{%- endif %} table
+    {{ relation.include(database=(not temporary), schema=(not temporary)) }}
+  as (
+    {{ sql }}
+  );
+
+{% endmacro %}
```

Comment:
love that these macros be broken into their own files! nit: can we make it `create_table_as.sql`? and do the same thing for `create_view_as` macro (`create_view_as.sql`)?

Reply:
yes, absolutely!